src/arith_uint256.cpp

```diff
@@ -173,8 +173,8 @@ unsigned int base_uint<BITS>::bits() const
 {
     for (int pos = WIDTH - 1; pos >= 0; pos--) {
         if (pn[pos]) {
-            for (int bits = 31; bits > 0; bits--) {
-                if (pn[pos] & 1 << bits)
+            for (size_t bits = 31; bits > 0; bits--) {
+                if pn[pos] & (1U << bits)
```

Comment:
Nit: I would prefer this if it were `if (pn[pos] & (1U << bits))`, but this is non-blocking.

Reply:
Huh, that's not even valid C++ as it is. Will fix.